_topic_map.yml

```diff
@@ -366,14 +366,18 @@ Topics:
 - Name: Ingress
   Dir: ingress
```

Comment:
I'd like to change above two lines to below:
```
 - Name: Understanding the Ingress Operator
   File: ingress-operator
 - Name: Configuring Routes
   File: configuring-routes
```
This is what I really want, I think "DNS operator", "Ingress Operator" and "Routes" are same level.

Reply:
Done.